airflow/providers/google/cloud/operators/vertex_ai/generative_model.py

```diff
@@ -666,7 +670,8 @@ def execute(self, context: Context):
         self.xcom_push(context, key="tuned_model_name", value=response.tuned_model_name)
         self.xcom_push(context, key="tuned_model_endpoint_name", value=response.tuned_model_endpoint_name)
 
-        return types_v1.TuningJob.to_dict(response)
+        # return types_v1.SupervisedTuningJob.to_dict(response) # SupervisedTuningJob object type not supported as of 9/2/2024
+        return response.tuned_model_endpoint_name
```

Comment:
I'd be happy for some clarifications:
1. Was `(Supervised)TuningJob` deprecated within a specific version of the Python SDK, or the API that it calls? It would be helpful if you could provide a source that state that.
2. The change from a dictionary to a string might be too drastic:
    -  Is there an easy way to pull the original response information?
    -  If not, maybe let's reduce it to a "minimized" dictionary that includes the endpoint_name using its original key? (i.e., something like `{"endpoint_name": response.tuned_model_endpoint_name}`)

Reply:
Thanks for the review.

1. It was not deprecated, the API response type changed in the last week. originally a TuningJob object, now a SupervisedTuningJob object. unfortunately the aiplatform.types package has not yet accommodated this change, therefore I cannot easily convert the custom type to a dictionary.
2. The operator is brand new, I created it within the past week. Not yet included in the latest release of airflow. So I think the change won't affect anyone at this point in time. I can still update the return type to be a minimized dictionary, however.